### DISCUSSION TITLE
Not connecting to steam.

### DISCUSSION THREAD

**Discussion by yahereisstinky** [WilliamVenner/gmpublisher#121] [Help]
I barely downloaded the application for gmod, however im getting an exclamation point and it says "Disconnected to Steam Server" 
what do?

**Comment by Chedski** [comment]
I had this issue too, try fully restarting steam (quit it from the taskbar icon or task manager)

**Comment by MrMarrant** [comment]
You have to be connected to steam (with the app) if you want it connect to your account

**Reply by JaydenUB** [reply]
How do I do it?

**Reply by btg611** [reply]
Get the app? From the website...
